tests/python/pants_test/engine/test_build_files.py

```diff
@@ -32,8 +34,8 @@ def test_empty(self):
     """Test that parsing an empty BUILD file results in an empty AddressFamily."""
     address_mapper = AddressMapper(JsonParser(TestTable()))
     af = run_rule(parse_address_family, address_mapper, Dir('/dev/null'), {
-        (Snapshot, PathGlobs): lambda _: Snapshot(DirectoryDigest('abc', 10), (File('/dev/null/BUILD'),)),
-        (FilesContent, DirectoryDigest): lambda _: FilesContent([FileContent('/dev/null/BUILD', b'')]),
+        (Snapshot, PathGlobs): lambda _: Snapshot(DirectoryDigest(text_type("abc"), 10), (File('/dev/null/BUILD'),)),
```

Comment:
Were these tests failing without these edits? Does this need a rebase to master?

Reply:
Py2 wasn’t failing, I don’t know about Py3. I had cleaned up the file because I realized we had more code than necessary. 

Ah that makes sense it might simply need to merge master. I wasn’t sure if it was intentional to revert the changes I had made.